CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt

```diff
@@ -0,0 +1,300 @@
+TM YB+1_S1 0 LB_RB+1_S1_BP1A 1580	CH0: W+1/RB1out/1 Backward 	CH1: W+1/RB2out/1 Backward 	CH2: W+1/RB1in/1 Backward 
```

Comment:
@fthyssen in my understanding one needs to: 
   * create a repo under https://github.com/cms-data/ named `CondTools/RPC/`
   * make a PR there with the data files 
   * wait (after it is approved) for the next IB and at that point remove the file from CMSSW (it will be taken directly from cvmfs

maybe @smuzaffar can further help